packages/examples/packages/wasm/snap.manifest.json

```diff
@@ -17,9 +17,6 @@
     }
   },
   "initialPermissions": {
-    "endowment:rpc": {
```

Comment:
?

Reply:
Yep, it's not detected and automatically removed because it's async. Will fix!